prometheus/prometheus.yml

```
@@ -41,4 +41,11 @@ scrape_configs:
     static_configs:
       - targets: ['node-exporter:9100']
-    scrape_interval: 5s 
\ No newline at end of file
+    scrape_interval: 5s
+
+  # Traefik 메트릭 추가
+  - job_name: 'traefik'
+    static_configs:
+      - targets: ['traefik:8080']
+    scrape_interval: 5s
+    metrics_path: '/metrics' 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)